firmware/quintuna/CRIT/src/io/io_canMsgQueues.h

```diff
@@ -36,8 +49,16 @@ namespace can1
     {
         app_canAlerts_CRIT_Warning_RxOverflow_set(false);
     }
+
+
 } // namespace can1
 
 static CanMsgQueue can1queue{ io_canRx_filterMessageId, can1::TxQueueOverflowCallback, can1::RxQueueOverflowCallback,
-                              can1::TxQueueOverflowClearCallback, can1::RxQueueOverflowClearCallback };
-} // namespace io
\ No newline at end of file
+                              can1::TxQueueOverflowClearCallback, can1::RxQueueOverflowClearCallback};
+
+void popRxMsgFromCan1Queue(){
+    hw::can::CanMsg rx_msg = can1queue.popRxMsgFromQueue();
```

Comment:
This function should probably return a cam message... let me talk to Edwin about this though